sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestAssetsHelper.cs

```diff
@@ -1,45 +1,55 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using Azure.DigitalTwins.Core.Serialization;
 
 namespace Azure.DigitalTwins.Core.Tests
 {
     public static class TestAssetsHelper
     {
+        private static readonly string NewLine = Environment.NewLine;
```

Comment:
really? :p

Reply:
I probably don't need this as we use it only in that function :)